public/requests/traces_request_handler.ts

```diff
@@ -191,7 +208,7 @@ const hitsToSpanDetailData = async (hits, colorMap) => {
     const duration = _.round(nanoToMilliSec(hit._source.durationInNanos), 2);
     const serviceName = _.get(hit, ['_source', 'serviceName']);
     const name = _.get(hit, '_source.name');
-    const error = hit._source['status.code'] ? 'Error' : '';
+    const error = hit._source['status.code'] === 2 ? ' \u26a0 Error' : '';
```

Comment:
what does the 2 represent in this expression?

Reply:
Currently data-prepper uses `status.code` = 0 to indicate unset, 1 is success and 2 is error. We only show error on UI if it's 2